release-notes/logstash-output-opensearch.release-notes-2.0.0.md

```diff
@@ -0,0 +1,15 @@
+## Version 2.0.0 Release Notes
+
+Compatible with OpenSearch 1.3.4, 2.1.0
+
+### Features
+
+* Configuration parameter `service_name` to specify the service name for Sig V4 signing. ([#168](https://github.com/opensearch-project/logstash-output-opensearch/pull/168))
```

Comment:
Per the AWS documentation this is named "Signature Version 4" or "SigV4". We should use one of those names, or maybe even document it as they have it "Signature Version 4 (SigV4)".

https://docs.aws.amazon.com/general/latest/gr/signature-version-4.html